app/models/user.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable func-names */
+/* eslint-disable no-unused-vars */
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        unique: true,
+        type: DataTypes.INTEGER
+      },
+      firstName: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      lastName: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      email: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        unique: true,
+        validate: {
```

Comment:
For convention, we not use validations in models. In SIGN-UP feature you must implement a validation for this